### DISCUSSION TITLE
Info: this list's size history

### DISCUSSION THREAD

**Discussion by StevenBlack** [FiltersHeroes/KAD#2158] [Pomysły]
Krystian @krystian3w this issue is for information only. I think you'll find this interesting.

I'm developing a variety of helper tools, still too early for prime-time, but this is a preliminary finding from analyzing my [hosts](https://github.com/StevenBlack/hosts) repo history.

The data [here](https://github.com/StevenBlack/hosts/blob/master/data/KADhosts/stats.out) has the granularity of my commits, whose frequency has varied over the years.

[This bash file](https://github.com/StevenBlack/hosts/blob/master/sourcestats.sh) is what I used on my repo to get data. This is easily adaptable for your use; feel free. I use Mathematica for the plot, but many other things can produce graphs like this from CSV data.

![KADhosts](https://user-images.githubusercontent.com/80144/181870280-cf019d1f-f3cd-4f8d-8d67-52f25259115b.png)

**Comment by admin** [comment]
That looks most at jump by injected CERT Polska Phishing into KAD sections.
